README.md

```diff
@@ -39,7 +39,7 @@ When using the CloudMicrophysics.jl inside your own project,
   and it's dependencies is to use the Julia built-in package manager
   (accessed by pressing `]` in the Julia REPL):
 
-```julia
+```bash
```

Comment:
Hi! I'm randomly lurking in CloudMicrophysics.jl, sorry (avoiding thesis writing!), but these should all stay as ```julia to indicate that you are typing this in a Julia REPL, not from the terminal directly.